### PR TITLE
Add Fluent Tutorial to Develop

### PR DESCRIPTION
**Pull request by jrossthomson**
### Submission Checklist

* [ ] Have you installed and run this change against pre-commit? (`pre-commit
  install`)
* [ ] Are all tests passing? (`make tests`)
* [ ] Have you written unit tests to cover this change?
* [ ] Is unit test coverage still above 80%?
* [ ] Have you updated all applicable documentation?
* [ ] Have you followed the guidelines in our Contributing document?

### CODE REVIEW

File: community/examples/fluent-tutorial.yaml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+          EOF
+
+        destination: /tmp/install-bashrc.sh
+      - type: shell
+        content: |
+          #!/bin/bash
+          mkdir /tmp/intel
+          cd /tmp/intel
+          wget https://registrationcenter-download.intel.com/akdlm/irc_nas/tec/13741/l_mpi_2018.4.274.tgz
+          tar xvf l_mpi_2018.4.274.tgz
```
Comment (heyealex) [inline community/examples/fluent-tutorial.yaml:66]: Pre-commit check: There's an extra space at the end of this line and on line 88

File: community/examples/fluent-tutorial.yaml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+          mkdir /tmp/intel
+          cd /tmp/intel
+          wget https://registrationcenter-download.intel.com/akdlm/irc_nas/tec/13741/l_mpi_2018.4.274.tgz
+          tar xvf l_mpi_2018.4.274.tgz 
+          cd l_mpi_2018.4.274
+          patch silent.cfg <<- "EOF1"
+          16:07:46.199308305 +0000
+          @@ -6,13 +6,13 @@
+           # snpat       - the serial number pattern (ABCD-01234567)
+
```
Comment (heyealex) [inline community/examples/fluent-tutorial.yaml:72]: Pre-commit check: The tab needs to be removed here and on lines 76, 79, and 83

File: community/examples/fluent-tutorial.yaml [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+      local_mount: /shared
+
+  - source: modules/compute/vm-instance
+    kind: terraform
+    id: compute_1
+    use:
+    - homefs
+    - sharefs
+    - startup
+
```
Comment (heyealex) [inline community/examples/fluent-tutorial.yaml:136]: nit: For consistency with the other modules, can this extra line be removed here and on line 156?

**Review by heyealex — CHANGES_REQUESTED** [review]
Thank you for contributing this Ross!

The pre-commit check flagged some a few lines in your yaml, I added comments to point out what needs to be updated to get past the check. I also included a couple minor change requests.

File: community/examples/fluent-tutorial.yaml [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: fluent-wl
+  region: us-central1
+  zone: us-central1-c
+
+deployment_groups:
+- group: primary
+  modules:
+  - source: ./modules/scripts/startup-script
```
[the remark this answers is not in the document]
Comment (jrossthomson) [reply community/examples/fluent-tutorial.yaml:28]: Done

File: community/examples/fluent-tutorial.yaml [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+      bandwidth_tier: "gvnic_enabled"
+      machine_type: c2-standard-8
+      instance_count: 1
+      name_prefix: fluent-login
+
+  - source: community/modules/scripts/wait-for-startup
+    kind: terraform
+    id: wait
+    settings:
+      instance_name: ((module.compute_1.name[0]))
```
[the remark this answers is not in the document]
Comment (jrossthomson) [reply community/examples/fluent-tutorial.yaml:169]: Done

File: community/examples/fluent-tutorial.yaml (hunk 2 above)
Comment (jrossthomson) [reply community/examples/fluent-tutorial.yaml:72]: Done

File: community/examples/fluent-tutorial.yaml (hunk 3 above)
Comment (jrossthomson) [reply community/examples/fluent-tutorial.yaml:136]: Done

**Comment by jrossthomson** [comment]
@heyealex PTAL.

File: examples/README.md [hunk 6]
```diff
@@ -729,10 +730,20 @@ python /var/tmp/qsim-example.py
 This blueprint provisions a simple cluster for use with a Simcenter StarCCM+
 tutorial.
 
-> **_NOTE:_** The tutorial has not yet been published.
+> The main tutorial is described on the [HPC Toolkit website](https://cloud.google.com/hpc-toolkit/docs/simcenter-star-ccm/run-workload).
 
 [starccm-tutorial.yaml]: ../community/examples/starccm-tutorial.yaml
 
+
```
Comment (heyealex) [inline examples/README.md:737]: One more flag from the pre-commit checks:
```
PyMarkdown...............................................................Failed
- hook id: pymarkdown
- exit code: 1

examples/README.md:737:1: MD012: Multiple consecutive blank lines [Expected: 1, Actual: 2] (no-multiple-blanks)
```

File: community/examples/fluent-tutorial.yaml [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+
+           # Continue with overwrite of existing installation directory, valid values are: {yes, no}
+           CONTINUE_WITH_INSTALLDIR_OVERWRITE=yes
+          EOF1
+
+          sudo ./install.sh -s silent.cfg
+
+        destination: /tmp/install-mpi.sh
+
+  - source: ./modules/scripts/startup-script
```
Comment (heyealex) [inline community/examples/fluent-tutorial.yaml:92]: I missed this one, but it should also be embedded: `modules/...` rather than `./modules/...`

**Review by heyealex — COMMENTED** [review]
Almost there! One more minor complaint from pre-commit and one more source that needs to be updated.

File: examples/README.md (hunk 6 above)
Comment (jrossthomson) [reply examples/README.md:737]: Fixed.

File: community/examples/fluent-tutorial.yaml (hunk 7 above)
Comment (jrossthomson) [reply community/examples/fluent-tutorial.yaml:92]: Got it.

**Comment by jrossthomson** [comment]
@heyealex PTAL

**Pull request merged**